supabase/migrations/20250522201752_add_sucker_group_id_aggregate_view.sql

```diff
@@ -0,0 +1,103 @@
+ALTER TABLE public.projects
+RENAME COLUMN redeem_voume_usd to redeem_volume_usd;
+ALTER TABLE public.projects
+ADD COLUMN IF NOT EXISTS sucker_group_id text;
+
+drop view if exists projects_aggregate;
+
+create view projects_aggregate as
+(
+  -- One project per suckerGroupId + array of chainIds
+  select distinct on (p.sucker_group_id)
+    p.id,
+    p.handle,
+    p.project_id,
+    p.pv,
+    p.sucker_group_id,
+    p.terminal,
+    p.deployer,
+    p.created_at,
+    p.name,
+    p.description,
+    p.logo_uri,
+    p.metadata_uri,
+    p.tags,
+    p.archived,
+    p.owner,
+    p.creator,
+    p.created_within_trending_window,
+    p.chain_id,
+    sub.chain_ids,
+    sub.contributors_count as contributors_count,
+    sub.current_balance as current_balance,
+    sub.payments_count as payments_count,
+    sub.nfts_minted_count as nfts_minted_count,
+    sub.redeem_count as redeem_count,
+    sub.redeem_volume as redeem_volume,
+    sub.redeem_volume_usd as redeem_volume_usd,
+    sub.trending_payments_count as trending_payments_count,
+    sub.trending_score as trending_score,
+    sub.trending_volume as trending_volume,
+    sub.volume as volume,
+    sub.volume_usd as volume_usd
+  from projects p
+  join (
+    select 
+      sucker_group_id, 
+      coalesce(array_agg(distinct chain_id) filter (where chain_id is not null), '{}') as chain_ids,
```

Comment:
we use `coalesce` to enforce non-null values at runtime (so we can manually type properties as non-null in the UI)